code/game/objects/items/weapons/design_disks/NeoTheology.dm

```diff
@@ -288,12 +264,13 @@
 		/datum/design/autolathe/cell/medium/high,
 	)
 
-/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/nt_counselor
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/nt/nt_counselor
 	disk_name = "NeoTheology Armory - Councelor PDW E"
 	icon_state = "neotheology"
 	spawn_tags = SPAWN_TAG_DESIGN_ADVANCED
 	rarity_value = 17
 	license = 12
+	spawn_blacklisted = TRUE
```

Comment:
undocumented change, the disk itself is super usefull when found in maint.

Reply:
Fuck I made a mistake, that was meant to be false.